### PR TITLE
Create Basic Components for Quick Create Section

### PR DESCRIPTION
**Pull request by knittingarch**
The components created here are placeholders for more robust design and
state.  Currently mocking up the components I will need for the MVP and
then will add state and interactivity as well as design.

The Quick Create section will be the top level way to create a task, but
users will also be able to click "more" if they want to add a set date or
frequency.  The defaults will be the current date and non-repeating.

Screen Shot 2019-10-04 at 5 50 49 PM

### CODE REVIEW

**Comment by knittingarch** [comment]
So I switched to master to check something and forgot to change back to this branch.  I messed up the fix so the files are listed under the second commit :/ Sigh.

File: src/AddTaskButton.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from "react"
+
+function AddTaskButton() {
+  return (
+    <button type="button">Add Task</button>
+  )
+}
+
+export default AddTaskButton
```
Comment (sidraval) [inline src/AddTaskButton.js]: Above on line 3, you can write `export default function AddTaskButton() { ....` export/define in the same declaration

File: src/LaunchModalButton.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+class LaunchModalButton extends React.Component {
```
Comment (sidraval) [inline src/LaunchModalButton.js]: This can be a functional component, because it's stateless

**Review by sidraval — APPROVED** [review]
Couple small comments, but looking good overall.

The comment about exporting/defining the functional component in the same declaration can be applied across most of your files.

One tool I see used in almost every JavaScript project these days is [prettier](https://github.com/prettier/prettier) for automated code formatting. I personally love it, because it takes all the formatting decisions out of the code.

File: src/LaunchModalButton.js (hunk 2 above)
Comment (knittingarch) [reply src/LaunchModalButton.js]: It's stateless now, but will have state shortly.  Is it better practice to write functional components and only convert once state is introduced?

**Pull request merged**